### DISCUSSION TITLE
Export Favourites

### DISCUSSION THREAD

**Discussion by wavard** [devemux86/cruiser#301] [Q&A] [answered]
Is it possible to export Favourites.  I want to copy Favourites from my phone to tablet. I have searched but cannot find an obvious way.

**Comment by devemux86** [comment, marked as answer]
Long press any favorite group for the menu and select:
- **Export**: export the selected group
- **Export all**: export all groups in a zip

**Comment by wavard** [comment]
Thank you

Regards

Walter
